.github/workflows/main.yml

```diff
@@ -26,26 +26,8 @@ jobs:
         opam_file: './coq-elpi.opam'
         coq_version: ${{ matrix.coq_version }}
         ocaml_version: ${{ matrix.ocaml_version }}
-        custom_script: |
-          startGroup Print opam config
-            opam config list; opam repo list; opam list
-          endGroup
-          startGroup Build dependencies
-            opam pin add -n -y -k path $PACKAGE $WORKDIR
-            opam update -y
-            opam install -y -j 4 $PACKAGE --deps-only
-          endGroup
-          startGroup List installed packages
-            opam list
-          endGroup
-          startGroup Build
-            opam install -y -v -j 4 $PACKAGE
-            opam list
-          endGroup
-          startGroup Test Hierarchy Builder
+        after_script: |
```

Comment:
Dear @erikmd if you have minute can you help me find the problem here?
I'm using the new feature of the action, but docker complains the script is empty.
I can't find the mistake.

Reply:
got it, I'll open an issue